data/views/redfish-1.0/redfish.1.0.0.accountservice.1.0.0.json

```diff
@@ -8,13 +8,13 @@
     "Name": "Account Service",
     "Status": {},
     "ServiceEnabled": true,
-    "MinPasswordLength": 8,
-    "MaxPasswordLength": 32,
```

Comment:
It is declared in the model: https://github.com/RackHD/on-core/blob/master/lib/models/local-user.js#L39